README.md

```diff
@@ -84,6 +93,69 @@ To use this module, add it to the modules array in the `config/config.js` file.
     },
 ```
 
+*Note:* This way you can only use one instance of each module. If you want to use multiple instances of the same module, you have to use the class based configuration.
+
+### Class based configuration
+
+Instead of using the module name, you can also use a class name for each page. This way you can have multiple instances of the same module on different pages.
+
+```js
+    {
+        module: "MMM-pages",
+        config: {
+            rotationTime: 1000 * 20, // rotate every 20 seconds
+            modules: [
+                ["page1"],  // class name for page 1
+                ["page2"],  // class name for page 2
+                ["page3"],  // class name for page 3
+            ],
+            fixed: ["fixed_page"],
+            hiddenPages: {
+                "screenSaver": ["screensaver_page"],
+                "admin": ["admin_page"],
+            }
+        }
+    },
+```
+
+You have to add the class name to the config of the module you want to show on a specific page. You can even add more than one class name to show a module istance on multiple pages.
```

Comment:
```suggestion
You have to add the class name to the config of the module you want to show on a specific page. You can even add more than one class name to show a module imstance on multiple pages.
```

Reply:
Oh! I found some other typos and just fixed them.